docs/release_notes.adoc

```diff
@@ -23,7 +23,7 @@ or used it as a delegate runner will not work anymore, e.g, `PowerMockRunnerDele
 
 === Misc
 
-- Add `@PendingFeatureIf` annotation (<<extensions.adoc#_pendingfeatureif,Docs>>)
+- Add `@PendingFeatureIf` annotation (<<extensions.adoc#pendingfeatureif,Docs>>)
```

Comment:
the underscore belongs to the link see http://spockframework.org/spock/docs/2.0-M2/extensions.html#_pendingfeatureif

Reply:
It doesn't work if clicked in the GitHub directly:
https://github.com/spockframework/spock/blob/master/docs/release_notes.adoc#misc
(there "_" is not used).

I didn't know it differs from the released version.

Reply:
The github renderer is not our target so I'm not too concerned about it.